UOP1_Project/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.7f1
-m_EditorVersionWithRevision: 2019.4.7f1 (e992b1a16e65)
+m_EditorVersion: 2019.4.9f1
```

Comment:
Regarding changing the Unity editor version, I've created a thread on the forum to discuss this https://forum.unity.com/threads/how-to-handle-different-unity-versions.980235/

Reply:
Please remove these changes from the PR, they aren't required and the Unity team will update the project periodically :)

Reply:
Please remove these changes from the PR, they aren't required and the Unity team will update the project periodically :)